tutorials/Python/Python_tutorial_solutions_python3.ipynb

```diff
@@ -58,17 +58,9 @@
   },
```

Comment:
Can we not delete this at this time? I would prefer having it around so we can backfill the solutions (mainly useful when jumpy between machines). We can remove it when we have our solutions created?

Reply:
It's not deleted, is just cleaned (i.e. execution outputs are removed); sorry, git diffs and notebooks really are horrible.

 It got caught into the `cleanup.sh` script.

Reply:
Sorry that's my bad :)